doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -157,31 +157,20 @@ the Bourne shell (sh)                ``. util/quickstart/setchplenv.sh``
 What's next?
 ------------
 
-=============================================== =========================
+=============================================== ==================================================
```

Comment:
If we dropped the header and instead used a short introduction before, like we do in the top-level `README.rst`, I think we can squeeze it down to 80-columns. Would you like me to do that?